### PR TITLE
Turn on flow in CI

### PR DESCRIPTION
**Pull request by cjb**
@keybase/react-hackers 

If this PR works, we should see a fail on the "push" build (because there were existing Flow failures in CI when I branched off master) and a success on the "pr" build (because Nojima has since fixed them).

### CODE REVIEW

**Comment by chrisnojima** [comment]
we could extract the version from flowconfig but :+1:

**Comment by cjb** [comment]
Changed to extract flow version from flowconfig now.

File: .travis.yml [hunk 1]
```diff
@@ -33,6 +33,7 @@ script:
   - if [ $TRAVIS_OS_NAME == 'linux' ] && [ $TRAVIS_BRANCH == 'master' ] && [ $TRAVIS_PULL_REQUEST == 'false' ]; then
       docker build -t kbclient .;
     fi
+  - cd ../shared && npm i --global flow-bin@`tail -n1 .flowconfig` && flow
```
Comment (chrisnojima) [inline .travis.yml]: this is fine, we could theoretically install https://github.com/npm/node-semver then do semver.valid() on it

**Comment by chrisnojima** [comment]
:+1: :shipit:

**Comment by cjb** [comment]
Cool, push failed and PR succeded, as predicted.  Merging in.

**Pull request merged**